tests/operator/operator.go

```diff
@@ -395,6 +395,13 @@ var _ = Describe("[sig-operator]Operator", Serial, decorators.SigOperator, func(
 		It("test VirtualMachineInstancesPerNode", func() {
 			newVirtualMachineInstancesPerNode := 10
 
+			var hypervisorDevice k8sv1.ResourceName
+			hypervisorDevice = services.KvmDevice
+			// if HyperVLayered feature gate is enabled, use the HyperV device instead of KVM
+			if checks.HasFeature(featuregate.HyperVLayered) {
+				hypervisorDevice = services.HyperVDevice
+			}
```

Comment:
I don't know if sharing is allowed but this is the same as `getHypervisorDevice()` in [tests/testsuite/fixture.go](https://github.com/kaizentm/kubevirt/pull/26/files#diff-6ebc78e9eda8b49e58438bdfc1c483bf68ea4eabb27c7b99ebb26dc3c48349b4).

Reply:
Ended up adding a new framework package to share some code